pkg/controller/mobiledeveloperconsole/mobiledeveloperconsole_controller.go

```diff
@@ -2,12 +2,13 @@ package mobiledeveloperconsole
 
 import (
 	"context"
-
 	mdcv1alpha1 "github.com/aerogear/mobile-developer-console-operator/pkg/apis/mdc/v1alpha1"
-
+	"github.com/aerogear/mobile-developer-console-operator/pkg/config"
```

Comment:
Thought: is that `config` package ever going to be useful other than here in this `mobiledeveloperconsole` package? I don't really know what the best practice for creating packages in Go is, just curious :)

Reply:
You mean as a external dependency? The pkg looks way too simple for that in my opinion

Reply:
> Thought: is that config package ever going to be useful other than here in this mobiledeveloperconsole package? I don't really know what the best practice for creating packages in Go is, just curious :)

I found that online :grin: 

I put it in a file within the same package first, but then there was a problem and I moved it to a separate package (I copied that config file from UPS and there the problem was contents of this file was not accessible by all controllers' packages. So, needed to move it to somewhere where all controllers were able to access it).